src/interfaces/ITimelockAlpha.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.8.10;
+
+interface ITimelockAlpha {
```

Comment:
Fetched this through cast interface

Reply:
I'm now wondering if we should change the name of this to `IPoolTogetherTimelock` to make it clear it's something PT specific. What do you think @alexkeating?

Reply:
Yes, I like that

Reply:
Made the change here https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/e1c936aa3e75314ad8e2d472e22f89f8e6c1ed9b